sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/CreateNewEventSteps.java

```diff
@@ -252,6 +254,23 @@ public CreateNewEventSteps(
               "Diseases statuses are not equal");
           softly.assertAll();
         });
+
+    When(
+        "^I create a new event with event identification source ([^\"]*)",
+        (String eventIdentificationSource) -> {
+          newEvent = collectEventUuid();
+          String timestamp = String.valueOf(System.currentTimeMillis());
+          webDriverHelpers.fillInWebElement(TITLE_INPUT, "EVENT_AUTOMATION" + timestamp);
+          selectEventIdentificationSource(eventIdentificationSource);
+          selectResponsibleRegion(RegionsValues.VoreingestellteBundeslander.getName());
+          selectResponsibleDistrict(DistrictsValues.VoreingestellterLandkreis.getName());
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(NEW_EVENT_CREATED_MESSAGE);
+          webDriverHelpers.clickOnWebElementBySelector(EVENT_TAB);
+          Assert.assertTrue(
```

Comment:
This assert should be treated as a separate step and be placed in EditEventSteps cause action that it performs in in that directory.